doc-content/enterprise-only/employee-rostering/er-availability-proc.adoc

```diff
@@ -13,13 +13,20 @@ In the top left part of the window, you can see the dates for which the roster i
 You can see the numbers of currently visible employees in the top right part of the browser window, for example, `1-10 of 34`. You can use the *<* and *>* buttons next to the numbers to display other employees in the list. 
 +
 . To create an availability entry for an employee, click empty space on the intersection of a date (column) and an employee (row). Initially, an `Unavailable` entry for the entire day is created.
-. To change an availability entry, double-click the entry. You can change the following settings:
+. To change an availability entry, click the entry. You can change the following settings:
 ** *From* and *To* date and time: The time span to which the availability entry applies.
 ** Status: you can select *Unavailable*, *Desired*, or *Undesired* status from a drop-down list.
 +
 To save the entry, click *Apply*.
-. To delete an availability entry, double-click the entry, then click *Delete availability*.
+. To delete an availability entry, click the entry, then click *Delete availability*.
++
+Alternatively, you can change or delete an availability entry by moving the mouse cursor over the entry. Several icons are displayed on the entry:
```

Comment:
You click the icons to change or delete the entry, so they are key to changing or deleting. But I will use "you can also" instead of "alternatively", and try to clarify about the icons.

I have my doubts regarding "hover", so I went to check the IBM Style Guide. It says "move the mouse pointer over" - explicit recommendation, page 345; it uses "hover" only in the context of "hover help". "Mouse cursor" is not exactly redundant (there is also a text entry cursor sometimes), but "mouse pointer" appears to be recommended,